Dutchskull.Aspire.AppHost.Unity3D/Dutchskull.Aspire.AppHost.Unity3D/UnityAspireExtensions.cs

```diff
@@ -12,11 +12,17 @@ namespace Aspire.Hosting;
 
 public static class UnityAspireExtensions
 {
+	public static IResourceBuilder<UnityProjectResource> AddUnityProject(
+		this IDistributedApplicationBuilder builder,
+		string name,
+		string projectPath
+		) => AddUnityProject(builder, name, projectPath, -1); 
+	
     public static IResourceBuilder<UnityProjectResource> AddUnityProject(
     this IDistributedApplicationBuilder builder,
     string name,
     string projectPath,
-    int? sceneIndex = null,
+    int sceneIndex,
```

Comment:
A user might just want to start what ever scene that is coming up when starting unity. Keeping this optional gives users this choice.